src/include/connection.h

```diff
@@ -369,8 +369,8 @@ struct __wt_connection_impl {
 	wt_thread_t	 sweep_tid;	/* Handle sweep thread */
 	int		 sweep_tid_set;	/* Handle sweep thread set */
 	WT_CONDVAR	*sweep_cond;	/* Handle sweep wait mutex */
-	time_t		 sweep_idle_time;/* Handle sweep idle time */
-	time_t		 sweep_interval;/* Handle sweep interval */
+	u_int		 sweep_idle_time;/* Handle sweep idle time */
```

Comment:
We're inconsistent about the types we use to store config values, but as it's coming from a config value, it might as well be an int64_t, that matches the underlying config size?